From d2cc93ca1d9af03edd06018425990f14c9da03d2 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Fri, 12 Sep 2014 11:48:47 -0400
Subject: [ethernet] i40e: Bump i40e/i40evf versions

Message-id: <1410522528-19793-10-git-send-email-sassmann@redhat.com>
Patchwork-id: 93941
O-Subject: [RHEL7.1 PATCH 09/10] i40e/i40evf: Bump i40e/i40evf versions
Bugzilla: 1140994
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>

From: Catherine Sullivan <catherine.sullivan@intel.com>

Bump i40e version to 1.0.11 and i40evf version to 1.0.5.

Change-ID: I63a60fa2efe82aae87a8a3095f43218db57d46ce
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Jim Young <jamesx.m.young@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit e966d5c612c08e04fe8ca1a87c2ba8403775b814)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index 44c5562..3aa6bcd 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -39,7 +39,7 @@ static const char i40e_driver_string[] =
 
 #define DRV_VERSION_MAJOR 1
 #define DRV_VERSION_MINOR 0
-#define DRV_VERSION_BUILD 4
+#define DRV_VERSION_BUILD 11
 #define DRV_VERSION __stringify(DRV_VERSION_MAJOR) "." \
       __stringify(DRV_VERSION_MINOR) "." \
       __stringify(DRV_VERSION_BUILD)    DRV_KERN
-- 
1.7.1